tests/buyers/views/test_buyers.py

```diff
@@ -2255,7 +2244,7 @@ def test_404_if_framework_is_not_live_or_expired(self):
     def test_get_responses(self):
         brief = mock.Mock()
 
-        with po(buyers, 'get_sorted_responses_for_brief') as m:
+        with mock.patch.object(buyers, 'get_sorted_responses_for_brief', autospec=True) as m:
```

Comment:
There's a fair amount of inconsistency in this PR between using po (which is mock.patch.object(autospec=True) and using the full thing inline. Any reason for/against each one? Not sure why we're changing it here.